extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js

```diff
@@ -67,7 +68,7 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.aol', [
 		};
 
 	function isEnabled() {
-		return adContext.get('bidders.aol');
+		return adContext.get('bidders.aol') && !babDetection.isBlocking();
```

Comment:
Wouldn't it be easier to disable all bidders based on this flag in adContext.js? Is it possible there?

Reply:
No, because in follow up ticket for InstartLogic we disabling only part of bidders.